src/Http/SessionHandlers/DatabaseHandler.php

```diff
@@ -33,7 +34,11 @@ public function write(string $id, string $data): bool
         $session->id = $id;
         $session->payload = $data;
         $session->last_activity = Carbon::now();
-        $session->user_id = auth()->user()?->id ?? null;
+        try {
+            $session->user_id = auth()->user()?->id ?? null;
+        } catch (Exception) {
+            // There might be some errors on early init
```

Comment:
Catch and handle the exception types you expect, not all exceptions. Exceptions aren't Pokemon, don't catch them all ;)